config.yaml

```diff
@@ -0,0 +1,38 @@
+options:
+  email:
+    type: string
+    description: Account email address
+  server:
+    type: string
+    description: Certificate authority server
+    default: "https://acme-v02.api.letsencrypt.org/directory"
+  aws_access_key_id:
+    type: string
+    default: ""
+    description: AWS Access Key ID
+  aws_secret_access_key:
+    type: string
+    default: ""
+    description: AWS Secret Access Key
```

Comment:
Should we have `defaults` for `aws_access_key_id` and `aws_secret_access_key`? Does this charm even make sense with no authentication?

Reply:
leftovers :)
✅